### DISCUSSION TITLE
Meaning of the class "Evidence"

### DISCUSSION THREAD

**Discussion by eprocurementontology** [eprocurementontology/eprocurementontology#486] [Ideas]
Claude Schmit asked for the class "evidence" : what does it mean ? some proving metadata about the "payment" ? so it should be linked to class "payment", not "contracting authority"

**Comment by JachymHercher** [comment]
(Don't overlook: the concept of evidence is very important for exclusion and selection criteria.)

**Comment by makxdekkers** [comment]
In the latest version of the conceptual model (to be provided ahead of the WG meeting of 24 May 2017), the class Evidence is indeed linked to Payment. The class Evidence will also be related Procurement Criterion and its subclasses, for example the way it is modelled in the Core Criterion and Core Evidence Vocabulary (see: https://joinup.ec.europa.eu/asset/criterion_evidence_cv/asset_release/core-criterion-and-core-evidence-vocabulary-v100).

**Comment by ocorcho** [comment]
It would be an interesting discussion to see whether the actual evidence (e.g., a pointer to a PDF with the payment from a bank or any pointer to a record in the payment system) is needed, or it is just enough to describe in the Payment class whether the invoice is paid and when it was paid. Example here from Zaragoza API: http://www.zaragoza.es/api/recurso/sector-publico/factura/62?rf=html
